lti_consumer/data.py

```diff
@@ -24,9 +26,19 @@ class Lti1p3ProctoringLaunchData:
         assessment message to after it has completed the proctoring setup and verification. This attribute is required
         if the message_type attribute of the Lti1p3LaunchData instance is "LtiStartProctoring". It is optional and
         unused otherwise.
+    * assessment_control_url (optional): The Platform URL that the Tool will send assessment control messages to.
```

Comment:
Did you have any thoughts about whether to add fields like `lti_1p3_proctoring_acs_enabled` and `lti_1p3_acs_actions` to the `LtiConfiguration` model? These data are specific to the integration, so they could be stored on the model, whereas the remainder of the data in this class can't be stored on the model because it isn't meant to be configurable.

That feels like overkill, but I also ask because that matches the pattern we have for the Advantage Services and the field for enabling proctoring on the LTI integration (`lti_1p3_proctoring_enabled`), so I'm mainly curious about our approach to when to add data to the model and when to send data through as part of this data class.

Reply:
@MichaelRoytman I started down the path of putting this on the model but pulled back and just chose to comment on the inconsistency. In our current implementation enabling ACS isn't optional or configurable (it must match what exams expects) so having this on the model doesn't make much sense right now.